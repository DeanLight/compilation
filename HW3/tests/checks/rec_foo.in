int foo(int x)
{
	return foo(x);
}
int foo2(int x)
{
	return foo2(x);
}
// no option for forawrds decleration
void main()
{
	foo(5);
}