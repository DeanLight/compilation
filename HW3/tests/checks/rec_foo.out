---end scope---
x INT -1
---end scope---
x INT -1
---end scope---
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
foo (INT)->INT 0
foo2 (INT)->INT 0
main ()->VOID 0
